diff --git a/assets/js/player.js b/assets/js/player.js
index 16bb2752..65c94aa0 100644
--- a/assets/js/player.js
+++ b/assets/js/player.js
@@ -20,9 +20,12 @@ var options = {
             'audioTrackButton',
             'qualitySelector',
             'playbackRateMenuButton',
+            'chromecastButton',
             'fullscreenToggle'
         ]
     },
+    techOrder: ['chromecast', 'html5'],
+    plugins: {chromecast: {}},
     html5: {
         preloadTextTracks: false,
         vhs: {
diff --git a/src/invidious/views/components/player_sources.ecr b/src/invidious/views/components/player_sources.ecr
index 9af3899c..dfb71f8f 100644
--- a/src/invidious/views/components/player_sources.ecr
+++ b/src/invidious/views/components/player_sources.ecr
@@ -13,6 +13,8 @@
 <script src="/videojs/videojs-markers/videojs-markers.js?v=<%= ASSET_COMMIT %>"></script>
 <script src="/videojs/videojs-share/videojs-share.js?v=<%= ASSET_COMMIT %>"></script>
 <script src="/videojs/videojs-vtt-thumbnails/videojs-vtt-thumbnails.js?v=<%= ASSET_COMMIT %>"></script>
+<script src="/videojs/silvermine-videojs-chromecast.min.js"></script>
+<script src="/js/cast_sender.js"></script>
 
 
 <% if params.annotations %>
